cost/azure/unused_sql_databases/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.2
+
+- fixed null error in script
```

Comment:
can we include details of what cause the error.  something like fix null for sku.name ?

Reply:
I have addressed this